foreshadow/steps/preparerstep.py

```diff
@@ -523,25 +522,27 @@ def inverse_transform(self, X, *args, **kwargs):
         self.check_process(X)
         return self._parallel_process.inverse_transform(X, *args, **kwargs)
 
-    @staticmethod
-    def _preparer_params():
-        init = getattr(
-            PreparerStep.__init__, "deprecated_original", PreparerStep.__init__
-        )
-        init_signature = signature(init)
-        # Consider the constructor parameters excluding 'self'
-        parameters = [
-            p
-            for p in init_signature.parameters.values()
-            if p.name != "self" and p.kind != p.VAR_KEYWORD
-        ]
-        return [p.name for p in parameters]
+    @classmethod
+    def _get_param_names(cls):
```

Comment:
Why do we need to override this method to include all parents params? The parent classes of PreparerStep are BaseEstimator and two Mixins whose parent is object. I'm not sure I'm following here.